src/Elliptic/BoundaryConditions/ApplyBoundaryCondition.hpp

```diff
@@ -0,0 +1,73 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <type_traits>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Domain/InterfaceHelpers.hpp"
+#include "Domain/Structure/Direction.hpp"
+#include "Domain/Tags.hpp"
+#include "Elliptic/BoundaryConditions/BoundaryCondition.hpp"
+#include "Elliptic/Utilities/ApplyAt.hpp"
+#include "Utilities/FakeVirtual.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace elliptic {
+/*!
+ * \brief Apply the `boundary_condition` to the `fields_and_fluxes` with
+ * arguments from interface tags in the DataBox.
+ *
+ * This functions assumes the arguments for the `boundary_condition` are stored
+ * in the DataBox in tags
+ * `domain::Tags::Interface<domain::Tags::BoundaryDirectionsInterior<Dim>,
+ * Tag>`. This may turn out not to be the most performant setup, so code that
```

Comment:
`performant` -> `efficient`. I don't know what `performant` means... Chandler Carruth gives a nice rant in his 2014 or 2015 cppcon talk about data structures and algorithms that "performant" doesn't really mean anything :)